scripts/deploy-ranger-plugin.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+################
+# Copy configs #
+################
+
+mkdir -p /etc/gcs/conf
+
+cp ranger-gcs-plugin/conf/* /etc/gcs/conf
+
+cp ./ranger-gcs-permission-check-service/conf/* /etc/gcs/conf
+
+###########################################
+# Resolve Ranger Server Hostname and Port #
+###########################################
+
```

Comment:
this won't work on non-Dataproc clusters. It is OK we only support Dataproc in this script, but you should explicitly point this out.